homeassistant/components/sensor/netdata.py

```diff
@@ -66,7 +66,7 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     try:
         response = requests.get(version_url, timeout=10)
         if not response.ok:
-            _LOGGER.error("Response status is '%s'", response.status_code)
```

Comment:
I think that we should keep it as error. We are only reporting it we were unable to fetch the data?